### DISCUSSION TITLE
It is possible to use A320NX LVAR ?

### DISCUSSION THREAD

**Discussion by JLD95** [maartentamboer/X-Touch-Mini-FS2020#102] [Q&A] [answered]
Hi,

at now, leds for SPLR armed and Parking brake indicator don't work with A20NX.
	  "simvar_led" : "BRAKE_PARKING_INDICATOR"
           "simvar_led": "SPOILERS_ARMED"

I would like to coonect leds to custom LVAR: it is possible ?

A32NX_PARK_BRAKE_LEVER_POS	0 | 1	R/W	Custom LVAR
A32NX_SPOILER_ARMED	        0 | 1	R	Custom LVAR

**Comment by Koseng** [comment]
Hi, 
yes. Just use "mobiflightsimvar_led". 

Examples:
``` json
"mobiflightsimvar_led": "(L:A32NX_AUTOPILOT_1_ACTIVE)"
"mobiflightsimvar_led": "(L:A32NX_SPOILERS_ARMED)"
"mobiflightsimvar_led": "(L:A32NX_AUTOTHRUST_STATUS) 2 == if"
"mobiflightsimvar_led": "(L:A32NX_FCU_APPR_MODE_ACTIVE)"
"mobiflightsimvar_led": "(L:A32NX_AUTOBRAKES_ARMED_MODE) 0 > if"
```

**Comment by JLD95** [comment, marked as answer]
Great, thank you very muchLe 23 avr. 2022 14:43, Koseng ***@***.***> a écrit :
Hi,
yes. Just use "mobiflightsimvar_led".
Examples:
"mobiflightsimvar_led": "(L:A32NX_AUTOPILOT_1_ACTIVE)"
"mobiflightsimvar_led": "(L:A32NX_SPOILERS_ARMED)"
"mobiflightsimvar_led": "(L:A32NX_AUTOTHRUST_STATUS) 2 == if"
"mobiflightsimvar_led": "(L:A32NX_FCU_APPR_MODE_ACTIVE)"
"mobiflightsimvar_led": "(L:A32NX_AUTOBRAKES_ARMED_MODE) 0 > if"

**Comment by Koseng** [comment]
Now with the new version v1.9.0 you can use "simvar_led" and "mobiflightsimvar_led" also with the encoders to turn on the whole led ring.

**Reply by JLD95** [reply]
It is possible to have an example to manage led ring?

**Reply by Koseng** [reply]
Attached my current file
[config_a320.zip](https://github.com/maartentamboer/X-Touch-Mini-FS2020/files/8549390/config_a320.zip)

**Reply by Koseng** [reply]
And the events.txt.
[events.txt](https://github.com/maartentamboer/X-Touch-Mini-FS2020/files/8549391/events.txt)

**Reply by Koseng** [reply]
And maybe also the overall config file.
[config.zip](https://github.com/maartentamboer/X-Touch-Mini-FS2020/files/8549411/config.zip)
